src/gc/gc.cpp

```diff
@@ -1973,7 +1973,7 @@ BOOL Aligned (size_t n)
 #define OBJECT_ALIGNMENT_OFFSET (sizeof(MethodTable *))
 
 #ifdef FEATURE_STRUCTALIGN
-#define MAX_STRUCTALIGN OS_PAGE_SIZE
+#define MAX_STRUCTALIGN GC_PAGE_SIZE
```

Comment:
I do not particularly like the GC_PAGE_SIZE name either.  GC_PAGE_SIZE is really something like GC_CARD_TABLE_SIZE or GC_4K right now.  If we agree on a better name I am happy to change it.